components/compliance-service/ingest/ingestic/mappings/comp-rep-date.go

```diff
@@ -270,6 +278,29 @@ var ComplianceRepDate = Mapping{
                     }
                   }
                 },
+                "waiver_data": {
+                  "type": "object",
+                  "properties": {
+                    "expiration_date": {
+                      "type": "text"
```

Comment:
should this be "type": "date" instead of "type": "text"?

Reply:
I thought about that. It comes from InSpec as a string(i.e. "1977-06-01") and we don't use it for ElasticSearch querying since it only matters when the scan is executed. After that, we rely on the `waiver_data` object. For this reason, I was thinking to keep it simple and avoid conversions.

Reply:
We talked about this in standup and decided to keep it as a string since we don't foresee a need in the future to treat the field as a date in the elasticsearch queries.